mobify/platform-scaffold
# Move user branch of the store

This PR moves all saved addresses into the user branch of the store. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1490
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/809

## Changes
- Update results used in merlins and SFCC connectors to use `receiveSavedAddresses` result
- Update selectors to use `storedAddresses` key name for getting addresses off the user branch of the store

## Todos

## How to test-drive this PR
- checkout the move-address-to-user-branch branch of the SDK
- run `npm run prod:build`
- run `npm link`
- cd into this repo
- run `npm link progressive-web-sdk`
- run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- sign in to an account with saved addresses
- view the address page
- the page should work as expected
- add an item to the cart and proceed to the shipping page in the checkout
- the page should work as expected (including showing the users saved addresses) 
- repeat using the SFCC connector/site
- everything should still work as expected
